### DISCUSSION TITLE
Send a Notification Before Making any Transaction

### DISCUSSION THREAD

**Discussion by mushfiqweb** [community/community#71000] [Other Feature Feedback, Questions, & Ideas]
### Like many other users, I also encounter this requirement with my bank.

This security feature means that I have to unblock my card manually every time I wish to make an online purchase. While this process provides a necessary layer of protection, it poses a challenge, especially when automated services like GitHub attempt to process transactions.

The issue arises because GitHub’s automated transactions can fail when the card is blocked, which is expected given the security protocols. To address this, I am seeking a solution that can provide me with timely notifications or specify the exact time of the transaction. Having this information beforehand would enable users to activate their cards in advance, ensuring seamless transactions and uninterrupted service.

**Is there a feature or notification system in place that could help users like me receive alerts about impending transactions? This proactive approach would not only enhance user experience but also assist in avoiding transaction failures due to card blocks.**

**Thank you for your attention to this matter. Your assistance in resolving this concern would be greatly appreciated.**

**Comment by Kaoser2023** [comment]
For some banks I guess this has been applied for extra security measures but as I can remember the scb cards did not have this layer .
